assets/src/domain/eventEditor/services/apollo/mutations/datetimes/useMutationHandler.ts

```diff
@@ -21,41 +21,44 @@ const useDatetimeMutationHandler = (): MutationHandler => {
 	const getMutationVariables = useMutationVariables();
 	const getOptimisticResponse = useOptimisticResponse();
 
+	const onUpdate = useCallback(
+		({ proxy, entity: datetime, input, mutationType }: any) => {
+			// Read the existing data from cache.
+			let data: DatetimesList;
+			try {
+				data = proxy.readQuery(options);
+			} catch (error) {
+				data = null;
+			}
+			const datetimes = data?.espressoDatetimes || DEFAULT_LIST_DATA;
+			const tickets = input?.tickets || [];
+
+			switch (mutationType) {
+				case MutationType.Create:
+					onCreateDatetime({ proxy, datetimes, datetime, tickets });
+					break;
+				case MutationType.Update:
+					onUpdateDatetime({ datetime, tickets });
+					break;
+				case MutationType.Delete:
```

Comment:
this function name is `onUpdate` and then inside there is this create and delete mutation;
maybe I don't get smth or maybe this function should have a different name

Reply:
That `onUpdate` is to handle apollo cache update.